l10n_th_address/__manifest__.py

```diff
@@ -32,6 +32,5 @@
     ],
     'installable': True,
     'auto_install': False,
+    'post_init_hook': '_update_data_noupdate',
```

Comment:
Sorry, I have mixed words. Yeah, that's it, for allowing to use outside OCA. Indeed, this is configurable, not at pylint-odoo level, but at https://github.com/OCA/maintainer-quality-tools. If we change the check there, anyone download that repository will find that error. Maybe we can check the target of the PR (if it's OCA/xxxx) and make it mandatory there, but I doubt it can be possible. @moylop260, any thought on this?